networking/p2p/src/test/java/tech/pegasys/artemis/networking/p2p/jvmlibp2p/gossip/AttestationTopicHandlerTest.java

```diff
@@ -65,6 +66,8 @@ public void setup() {
   @Test
   public void onNewAttestation() {
```

Comment:
It's probably worth adding another test that confirms that attestations for the wrong committee are ignored.

Reply:
doing that now.

Reply:
done